src/main/java/yapmeister/task/TaskList.java

```diff
@@ -0,0 +1,76 @@
+package yapmeister.task;
+
+import java.util.ArrayList;
+import java.util.function.Predicate;
+
+/**
+ * Represents a TaskList that holds tasks
+ * @author BlazeChron
+ */
+public class TaskList {
+    private ArrayList<Task> tasks;
+
+    /**
+     * Creates a null TaskList.
+     */
+    public TaskList() {
+        this.tasks = null;
+    }
+
+    /**
+     * Creates a TaskList with the specified ArrayList.
+     * @param tasks Given ArrayList to hold tasks
+     */
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    /**
+     * Adds task to the TaskList.
+     * @param t Task to be added.
+     */
+    public void addTask(Task t) {
+        this.tasks.add(t);
```

Comment:
By the coding quality naming conventions, names that are 'too short' are not preferable, so you can change it to be task